test/tests/functions/integer.json

```diff
@@ -54,7 +54,7 @@
     },
     {
       "src": ".local $sel = {1 :integer select=exact} .match $sel 1 {{literal select {$sel}}} * {{OTHER}}",
-      "exp": "literal select {1}"
+      "exp": "literal select 1"
```

Comment:
This is right, although don't we try to avoid depending on the specific output of a formatter? Perhaps make the pattern `{{literal select 1}}`

Reply:
No, we need to validate that the formatting of `$sel` emits no error here, unlike `$bad` in the next two tests.